saleor/graphql/account/mutations.py

```diff
@@ -148,6 +152,32 @@ class Meta:
         model = models.User
 
 
+class LoggedCustomerUpdate(CustomerCreate):
+    class Arguments:
+        id = graphene.ID(
+            description='ID of a customer to update.', required=True)
+        input = UserAddressInput(
+            description='Fields required to update logged in customer.',
+            required=True)
+
+    class Meta:
+        description = 'Updates an existing logged in customer.'
+        exclude = ['password']
+        model = models.User
+
+    @classmethod
+    def user_is_allowed(cls, user, input):
+        id = graphene.Node.to_global_id('User', user.id)
+        return user.has_perm('account.manage_users') or id == input['id']
+
+    @classmethod
+    def mutate(cls, root, info, **data):
+        if data['id'] == '':
```

Comment:
Can we use `if not data['id']`, or is there a case when ID will evaluate to False?
Also, do we even need to ask for `id` if this mutation can be used without it?